tests/ci/create_test_entries.py

```diff
@@ -66,54 +153,152 @@ def _insert_entry(name, relpath, version, owner_type, owner, description):
         old_location=old_location,
         copy=(not make_sym_link),
         is_dummy=is_dummy,
+        execution_id=execution_id,
     )
 
     assert new_id is not None, "Trying to create a dataset that already exists"
     print(f"Created dataset entry with id {new_id}")
 
+    return new_id
+
 
-# Make some test entries. These will be dummy entries, copying no actual data.
-_insert_entry(
-    "DESC dataset 1",
+# Test set 1
+# - Auto create name for us
+_insert_dataset_entry(
     "DESC/datasets/my_first_dataset",
     "0.0.1",
     "user",
     None,
     "This is my first DESC dataset",
 )
 
-_insert_entry(
-    "DESC dataset 1",
-    "DESC/datasets/my_first_dataset_v2_minor_upgrade",
-    "minor",
+# Test set 2
+# - Manual name
+_insert_dataset_entry(
+    "DESC/datasets/my_first_named_dataset",
+    "0.0.1",
     "user",
     None,
-    "This is my first DESC dataset (minor version update)",
+    "This is my first named DESC dataset",
+    name="named_dataset",
 )
 
-_insert_entry(
-    "DESC dataset 2",
-    "DESC/datasets/my_second_dataset",
+# Test set 3
+# - Test version bumping
+_insert_dataset_entry(
+    "DESC/datasets/bump_dataset",
```

Comment:
A very minor point but - could we change the names to `bumped_dataset`, `bumped_dataset_2`, etc.?